deps.edn

```diff
@@ -2,17 +2,17 @@
  :deps {org.clojure/clojure {:mvn/version "1.11.1" :mvn/scope "provided"}}
 
  :aliases
- {:build {:deps {io.github.clojure/tools.build {:git/tag "v0.9.6"
+ {:build {:deps {io.github.clojure/tools.build {:git/tag "v0.10.0"
                                                 :git/sha "8e78bcc"}
```

Comment:
Don't forget to change the sha: `3a2c484`

https://github.com/clojure/tools.build/releases/tag/v0.10.0

Reply:
Good catch, thank you!